src/fmelt.c

```diff
@@ -595,9 +595,12 @@ SEXP getvarcols(SEXP DT, SEXP dtnames, Rboolean varfactor, Rboolean verbose, str
   if (data->lvalues==1 && length(VECTOR_ELT(data->valuecols, 0)) != data->lmax)
     error(_("Internal error: fmelt.c:getvarcols %d %d"), length(VECTOR_ELT(data->valuecols, 0)), data->lmax);  // # nocov
   if (isNull(data->variable_table)) {
+    if (data->lvalues == 1 & data->measure_is_list) {
```

Comment:
`-Wparentheses` wants this to use parens to emphasize operator precedence

```suggestion
    if ((data->lvalues == 1) & data->measure_is_list) {
```